packages/slice-machine/lib/models/server/ServerState.ts

```diff
@@ -16,17 +16,11 @@ export interface ConfigErrors {
 export interface ServerState {
```

Comment:
I hope the Preview PR doesn't use stuff that we removed here :x

Reply:
There will be conflicts because the AppPayload was renamed and changed in the PreviewBranch. But I think I can handle the conflicts :)